### PR TITLE
OPSEXP-1951: refactor zepellin chart inclusion

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1951

### CODE REVIEW

**Comment by alxgomz** [comment]
Due to https://alfresco.atlassian.net/browse/MNT-23457 I had to use a workaround to test zeppelin deployment works ok:
https://github.com/Alfresco/acs-deployment/actions/runs/4295568077

Now reverting that workaround before review requests

File: helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin/templates/NOTES.txt [hunk 1]
```diff
@@ -8,5 +7,4 @@ You can access Alfresco Insight Zeppelin using:
 If you have a specific DNS address for the cluster please run the following commands to get the application paths and configure ACS:
 
 helm upgrade --reuse-values {{ .Release.Name }} --set externalProtocol="http" --set externalHost="domain.com" --set externalPort="80" alfresco/alfresco-content-services
-{{ end }}
 {{- end }}
```
Comment (gionn) [inline helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin/templates/NOTES.txt:10]: does it make sense after removing the enabled logic? can we get rid of the entire file?

Comment (alxgomz) [reply helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin/templates/NOTES.txt:10]: Looks like that file actually never was included only the NOTES from the main charts are rendered. Will remove and maend the root NOTES

Comment (alxgomz) [reply helm/alfresco-content-services/charts/alfresco-search/charts/alfresco-insight-zeppelin/templates/NOTES.txt]: and let's try ith a bit of fun :)

**Review by upandey123 — APPROVED** [review]

File: helm/alfresco-content-services/templates/NOTES.txt [hunk 2]
```diff
@@ -5,17 +5,36 @@
...
+ 0@@@@@BM"",,,,,,,_   +pppppp@@N@@@@@@E
+ '@@@P_mBNNNNNNBN",@ Ew`TNNNNNNB@@@@@@"
+  `0"/BNNNNNNBP`,0N@ 0B@p_TBNNNNNN@@R`
+     BBBBBBBE`,BNNN@ 0NNB@b,`"MMM^"
+    !@@@@@@@ ANNNNB@ 0NNNNN@@@BBBN@L
+     T@@@@@E NNNNNN@L'BNNNNb@@@@@@P
+      '0@@@@ 0NNBN@@@p_TBNBN@@@@B"
+         `""  0@@@@@@@@b,_`"""`
+               `T0@@@@BP`
+
```
Comment (gionn) [inline helm/alfresco-content-services/templates/NOTES.txt:26]: 🥇

**Review by gionn — APPROVED** [review]

**Review by slohe1 — APPROVED** [review]

**Review by upandey123 — APPROVED** [review]

**Pull request merged**